src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,8 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Resources.ResourceReader" feature="System.Resources.ResourceManager.CustomResourceTypes.IsSupported" featurevalue="false">
```

Comment:
Nit: It should be either `CustomResourceTypes.AreSupported` or `CustomResourceType.IsSupported`.